src/main/java/catan/services/impl/PlayServiceImpl.java

```diff
@@ -138,6 +149,17 @@ private void throwDice(GameBean game) {
         game.setDiceThrown(true);
     }
 
+    private void buyCard(GameUserBean gameUser, GameBean game, Map<String, String> returnedParams) throws PlayException, GameException {
```

Comment:
dont think so. we had a precedents earlier when we changed some details is stories in the air. BTW, actually story development did not started yet. Alex just assigned the task for himself one week ago but made nothing, he said about it. Branch even not created.